src/Symfony/Component/VarDumper/VarDumper.php

```diff
@@ -29,7 +29,7 @@ public static function dump($var)
     {
         if (null === self::$handler) {
             $cloner = new VarCloner();
-            $dumper = in_array(PHP_SAPI, array('cli', 'phpdbg')) ? new CliDumper() : new HtmlDumper();
+            $dumper = in_array(PHP_SAPI, array('cli', 'phpdbg')) && !($_SERVER['VAR_DUMPER_HTML'] ?? false) ? new CliDumper() : new HtmlDumper();
```

Comment:
it could work both ways: forcing CLI instead of HTML, for API responses

Reply:
done